src/app/globals.css

```
@@ -1275,6 +1276,13 @@ details[data-collapsible][open] > summary [data-chevron] {
     0 0 12px var(--color-card-glow-shadow);
 }
 
+/* Disc chips start at nav size and shrink via `--chip-fs` (set after mount)
+ * so C1–C3 / Pochven / Drifter stay on one line instead of wrapping. */
+.map-node-disc [data-chain-node-classification] {
+  --chip-fs: var(--text-nav);
+  font-size: var(--chip-fs);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)